rpcclient/infrastructure.go

```diff
@@ -79,6 +79,19 @@ var (
 	ErrRequestCanceled = errors.New("request was canceled by the caller")
 )
 
+// AuthType specifies the RPC authorization method.
+//
+// This defaults to [AuthTypeBasic] if not specified.
```

Comment:
It looks like you put this on the wrong one.  This is the type definition so it "defines the supported RPC authorization methods".

The is all describing what happens for the configuration field in the `ConnConfig` struct.

Reply:
I don't think my comment was clear.

You fixed the "defines" part, but all of the rest of this needs to be moved to the field on the struct, because that is what it applies to, not the type.